source/ed/data.cpp

```diff
@@ -621,58 +496,20 @@ create_shape(const nt::GeographicalCoord& from,
     return simplified;
 }
 
-void Data::build_journey_pattern_points(){
-    auto logger = log4cplus::Logger::getInstance("log");
-    LOG4CPLUS_TRACE(logger, "Construct journey_pattern points");
-    std::map<std::string, ed::types::JourneyPatternPoint*> journey_pattern_point_map;
-
-    size_t nb_stop_time = 0;
-    for(types::VehicleJourney * vj : this->vehicle_journeys){
+void Data::build_shape_from_prev() {
+    for (types::VehicleJourney* vj: vehicle_journeys) {
```

Comment:
Technically, you can, but as you'll modify objects using this path, I prefer letting non const